contracts/ainft/AINFTBaseV2.sol

```diff
@@ -0,0 +1,210 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
Could you add some notes on the differences between v1 and v2 for the users?

Reply:
I added comments for contract information!
(ref. https://github.com/chiru-labs/ERC721A/blob/main/contracts/ERC721A.sol)

Reply:
Yes. I added comments for AINFTBaseV2 contract! (ref. [ERC721A's comments](https://github.com/chiru-labs/ERC721A/blob/main/contracts/ERC721A.sol))